packages/@webex/plugin-meetings/README.md

```diff
@@ -572,6 +605,7 @@ meeting.inMeetingActions.get();
   canRaiseHand: boolean,
   canLowerAllHands: boolean,
   canLowerSomeoneElsesHand: boolean,
+  canEndMeeting: boolean,
```

Comment:
question: Why has this file got so many modifications, when there are only 2 lines that actually add `canEndMeeting`? I suspect it's some auto formatting, and it's probably fine, was just surprised to see so many changes here!